_data/materials.yaml

```diff
@@ -2,128 +2,128 @@
 - id: 5
-  date: 10/4/2021
+  date: 2/28/2022
   beginner-name: Networking 101
   beginner-slides: https://docs.google.com/presentation/d/1DCN2fViHjAS7PoS98dsVC-LNwPN_jvaAKIuVFaRJrM0/edit?usp=sharing
   beginner-video: https://youtu.be/cPkLsSFKwPM
   advanced-name: Processes and Services
   advanced-slides: https://docs.google.com/presentation/d/1sv5Bxw9Fet9utjFU8xGDntawLVmBOT9YpCt9mJ2V6Dg/edit
   advanced-video: https://youtu.be/AS293JoAr8s
-  beginner-solutions: https://docs.google.com/document/d/1ltsmW3Tg_VRN9Slt9nSYAxZtqMAFQvlsO6NsJjag6m0/edit?usp=sharing
-  advanced-solutions: https://docs.google.com/document/d/1gCotUApMFWvhGXpkXozwOkPhd4AGYqp7jmyyutbkIIM/edit?usp=sharing
-  labdue: Sat. 10/9
-  hidelink: false
+  #beginner-solutions: https://docs.google.com/document/d/1ltsmW3Tg_VRN9Slt9nSYAxZtqMAFQvlsO6NsJjag6m0/edit?usp=sharing
+  #advanced-solutions: https://docs.google.com/document/d/1gCotUApMFWvhGXpkXozwOkPhd4AGYqp7jmyyutbkIIM/edit?usp=sharing
+  labdue: Sat. 3/5
+  hidelink: true
 
 - id: 6
-  date: 10/11/2021
+  date: 3/7/2022
   beginner-name: Processes
   beginner-slides: https://docs.google.com/presentation/d/1AS1BYX8jdBQGBviBGgcZPREIePrsgkT7pc-9Wk_RkS0/edit?usp=sharing
   beginner-video: https://youtu.be/Z_q1zkeWEFs
   advanced-name: Networking 102
   advanced-slides: https://docs.google.com/presentation/d/14XXyjUxUohINsg93qD-8SHP1mPpTTnBoZ3NHmMVTJqY/edit?usp=sharing
   advanced-video: https://youtu.be/DPKRivN9ga4
-  beginner-solutions: https://docs.google.com/document/d/1LQQzsYmZrplLFdCykMwBcXJwb7qu35ZqlXJogonhNCI/edit?usp=sharing
-  advanced-solutions: https://docs.google.com/document/d/1D9V_4VB4vu2FYRrQgID2m98ak2qEzXAuqWr2hSUoz1U/edit?usp=sharing
-  labdue: Sat. 10/16
-  hidelink: false
+  #beginner-solutions: https://docs.google.com/document/d/1LQQzsYmZrplLFdCykMwBcXJwb7qu35ZqlXJogonhNCI/edit?usp=sharing
+  #advanced-solutions: https://docs.google.com/document/d/1D9V_4VB4vu2FYRrQgID2m98ak2qEzXAuqWr2hSUoz1U/edit?usp=sharing
+  labdue: Sat. 3/12
+  hidelink: true
 
 - id: 7
-  date: 10/18/2021
+  date: 3/14/2022
   beginner-name: Services
   beginner-slides: https://docs.google.com/presentation/d/1N67saQd3NWrBLrk1oEOE2UdLa4Ny_qjDBUAZQolXsUE/edit?usp=sharing
   beginner-video: https://youtu.be/y3mIzYtCPeY
   advanced-name: Networked Services
   advanced-slides: https://docs.google.com/presentation/d/11nMxy5iJTS3YDBLqT3O5i8cbbRdtS2Hxf4QRbHJiGBo/edit
   advanced-video: https://youtu.be/5clc5xi5FDo
-  beginner-solutions: https://docs.google.com/document/d/1u5KV_LIfUbQHvPbspq_ejHxO1LsxHskMIlcHVgi0AhQ/edit?usp=sharing
-  advanced-solutions: https://docs.google.com/document/d/1YiSOMMNl_H5XNmvEqo53tCQ8xHFurqBMkIq6dWJ7ExE/edit?usp=sharing
-  labdue: Sat. 10/23
-  hidelink: false
+  #beginner-solutions: https://docs.google.com/document/d/1u5KV_LIfUbQHvPbspq_ejHxO1LsxHskMIlcHVgi0AhQ/edit?usp=sharing
+  #advanced-solutions: https://docs.google.com/document/d/1YiSOMMNl_H5XNmvEqo53tCQ8xHFurqBMkIq6dWJ7ExE/edit?usp=sharing
+  labdue: Sat. 3/19
```

Comment:
maybe we should make this lab due after spring break vs before?

Reply:
Would be a good idea. Were you thinking Saturday or Sunday after break?

Reply:
yeah that makes sense, I'll shift it to after the break. Maybe keep it Saturday to be consistent with the other due dates?